client/src/components/AllEntries.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react'
+
+export default function AllEntries() {
+    fetch("/allEntries")
+    .then((res) => res.json())
+    .then((entry) => {
+      entry.forEach((element) => {
+        // container
+        let Cont = document.createElement("div");
+        Cont.id = element._id;
+        Cont.className = "listing-cont";
```

Comment:
This could be written out as `<div id={element._id} className="listing-cont"></div>`